### PR TITLE
Move DataManager to CleintData

### PR DESCRIPTION
**Pull request by dimitry-**
Test: builds

### CODE REVIEW

File: src/ClientData/include/ClientData/DataManager.h [hunk 1]
```diff
@@ -127,18 +128,18 @@ class DataManager final {
 
  private:
   const std::thread::id main_thread_id_;
-  orbit_client_data::FunctionInfoSet selected_functions_;
-  absl::flat_hash_set<uint64_t> visible_function_ids_;
+  FunctionInfoSet selected_functions_;
+  absl::lts_2020_09_23::flat_hash_set<uint64_t> visible_function_ids_;
```
Comment (karupayun) [inline src/ClientData/include/ClientData/DataManager.h:132]: Is this intended? Could you explain me what are the advantages of this?

**Review by karupayun — APPROVED** [review]
LGTM!

File: src/ClientData/include/ClientData/DataManager.h (hunk 1 above)
Comment (dimitry-) [reply src/ClientData/include/ClientData/DataManager.h:132]: ops... not intended.

**Review by florian-kuebler — APPROVED** [review]
LGTM, thanks!

**Pull request merged**